Documentation/operations/troubleshooting.rst

```diff
@@ -222,27 +222,20 @@ simultaneously and aggregate the results. See :ref:`hubble_setup` to enable
 Hubble Relay if it is not yet enabled and install the Hubble CLI on your local
 machine.
 
-You may access the Hubble Relay service by port-forwarding it locally:
-
-.. code-block:: shell-session
-
-   kubectl -n kube-system port-forward service/hubble-relay --address 0.0.0.0 --address :: 4245:80
-
-This will forward the Hubble Relay service port (``80``) to your local machine
-on port ``4245`` on all of it's IP addresses.
+.. include:: /observability/hubble/port-forward.rst
 
 You can verify that Hubble Relay can be reached by using the Hubble CLI and
 running the following command from your local machine:
 
 .. code-block:: shell-session
 
-   hubble status
+   hubble status -P
 
 This command should return an output similar to the following:
 
-::
+.. code-block:: shell-session
 
-   Healthcheck (via localhost:4245): Ok
+   Healthcheck (via 127.0.0.1:4245): Ok
    Current/Max Flows: 16380/16380 (100.00%)
    Flows/s: 46.19
    Connected Nodes: 4/4
```

Comment:
Replacing the `::` is not necessary here, because you don't have a command in the block. (But it's probably not really an issue, either.)

Reply:
Ha yeah right makes sense, I reverted it